server/lib/amplitude.js

```diff
@@ -124,12 +123,11 @@ const transform = initialize(SERVICES, EVENTS, FUZZY_EVENTS);
 
 module.exports = receiveEvent;
 
-function receiveEvent (event, request, data) {
+function receiveEvent (event, request, data, location) {
```

Comment:
nit: it's strange that location is not part of `data`, feels like we are just gonna keep adding arguments here?